File: app/src/main/java/com/jggdevelopment/simpleweather/models/AlertDatum.java
package com.jggdevelopment.simpleweather.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AlertDatum {

    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("time")
    @Expose
    private int time;
    @SerializedName("expires")
    @Expose
    private int expiryTime;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("uri")
    @Expose
    private String uri;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public int getTime() {
        return time;
    }

    public void setTime(int time) {
        this.time = time;
    }

    public int getExpiryTime() {
        return expiryTime;
    }

    public void setExpiryTime(int expiryTime) {
        this.expiryTime = expiryTime;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getUri() {
        return uri;
    }

    public void setUri(String uri) {
        this.uri = uri;
    }
}

File: app/src/main/java/com/jggdevelopment/simpleweather/models/AlertDatum.kt
package com.jggdevelopment.simpleweather.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AlertDatum {

    @SerializedName("title")
    @Expose
    var title: String? = null
    @SerializedName("time")
    @Expose
    var time: Int = 0
    @SerializedName("expires")
    @Expose
    var expiryTime: Int = 0
    @SerializedName("description")
    @Expose
    var description: String? = null
    @SerializedName("uri")
    @Expose
    var uri: String? = null
}
